docs/changelogs/v0.18.md

```diff
@@ -130,7 +130,7 @@ since Kubo 0.13, but in this release it will also include the size column.
 ##### WebTransport enabled by default
 [WebTransport](https://docs.libp2p.io/concepts/transports/webtransport/) is a new libp2p transport that [was introduced in v0.16](https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.16.md#-webtransport-new-experimental-transport) that is based on top of QUIC and HTTP3.
 
-This allows browsers to contact Kubo nodes, so now instead of just serving requests for other system level applicative nodes, you can also serve requests directly to a browser.
+This allows browsers based nodes to contact Kubo nodes, so now instead of just serving requests for other system level applicative nodes, you can also serve requests directly to a node running inside a browser page.
```

Comment:
```suggestion
This allows browser-based nodes to contact Kubo nodes, so now instead of just serving requests for other system-level application nodes, you can also serve requests directly to a node running inside a browser page.
```